client/components/Scenario/ContentSlide.jsx

```diff
@@ -41,6 +43,11 @@ class ContentSlide extends React.Component {
         const isSkip = true;
         const value = '';
 
+        if (!this.props.run) {
+            alert('Slides cannot be skipped in Preview');
```

Comment:
Do we need an alert? We didn't have one for the "Next" button in preview, I think it'll be clear enough in context why they can't skip in preview.